src/index.js

```diff
@@ -1,18 +1,7 @@
 /* eslint-disable no-unused-vars */
-import React from 'react';
-import ReactDOM from '../node_modules/react-dom/client.js';
+import ReactDOM from 'react-dom/client';
+import Calculator from './components/Calculator.js';
 import './index.css';
-import App from './App.js';
-import reportWebVitals from './reportWebVitals.js';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-);
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+root.render(<Calculator />);
```

Comment:
- [ ] Great job working on the UI design but I think here you should just keep the app file iside your `index.js` and keep it and you will work with your `app.js` file and you will import all your components there and just the index.js has to compile `app.js` that's the struture of React and it is very important to follow this flow 😊💯💪🏻🚀 so kindly make a new file called app.js and import your `Calculator.js` inside it. ☺💪🏻💯💯💯



![image](https://user-images.githubusercontent.com/58553711/197653665-d039bbd2-5a4c-4e79-9af0-ea976b813021.png)